Use launcher script instead of directly running from the jar file.

--- languagetool-1.29/plugin/LanguageTool.vim
+++ languagetool-1.29/plugin/LanguageTool.vim
@@ -162,23 +162,6 @@
       endif
     endif
   endif
-
-  let s:languagetool_jar = exists("g:languagetool_jar")
-  \ ? g:languagetool_jar
-  \ : $HOME . '/languagetool/languagetool-commandline.jar'
-
-  if !filereadable(s:languagetool_jar)
-    " Hmmm, can't find the jar file.  Try again with expand() in case user
-    " set it up as: let g:languagetool_jar = '$HOME/languagetool-commandline.jar'
-    let l:languagetool_jar = expand(s:languagetool_jar)
-    if !filereadable(expand(l:languagetool_jar))
-      echomsg "LanguageTool cannot be found at: " . s:languagetool_jar
-      echomsg "You need to install LanguageTool and/or set up g:languagetool_jar"
-      echomsg "to indicate the location of the languagetool-commandline.jar file."
-      return -1
-    endif
-    let s:languagetool_jar = l:languagetool_jar
-  endif
   return 0
 endfunction
 
@@ -242,8 +225,7 @@
   let l:range = a:line1 . ',' . a:line2
   silent exe l:range . 'w!' . l:tmpfilename
 
-  let l:languagetool_cmd = 'java'
-  \ . ' -jar '  . s:languagetool_jar
+  let l:languagetool_cmd = 'languagetool'
   \ . ' -c '    . s:languagetool_encoding
   \ . (empty(s:languagetool_disable_rules) ? '' : ' -d '.s:languagetool_disable_rules)
   \ . ' -l '    . s:languagetool_lang
